app/forms/apo_form.rb

```diff
@@ -101,6 +109,14 @@ def options_for_use_license_type(current_value)
     end.compact # the options block will produce nils for unused deprecated entries, compact will get rid of them
   end
 
+  def check_default_object_rights
+    default_access = model.administrative&.defaultAccess
```

Comment:
How about handling the case where default_access is `nil` first? Then you can get rid of the rest of the `&.` in this method.

Reply:
Done, thanks!